configs/milvus.yaml

```diff
@@ -133,11 +133,17 @@ minio:
   # aliyun (ecs): https://www.alibabacloud.com/help/en/elastic-compute-service/latest/attach-an-instance-ram-role
   useIAM: false
   # Cloud Provider of S3. Supports: "aws", "gcp", "aliyun".
+  # Cloud Provider of Google Cloud Storage. Supports: "gcpnative".
   # You can use "aws" for other cloud provider supports S3 API with signature v4, e.g.: minio
   # You can use "gcp" for other cloud provider supports S3 API with signature v2
   # You can use "aliyun" for other cloud provider uses virtual host style bucket
+  # You can use "gcpnative" for the Google Cloud Platform provider. Uses service account credentials
+  # for authentication.
   # When useIAM enabled, only "aws", "gcp", "aliyun" is supported for now
   cloudProvider: aws
+  # The JSON content contains the gcs service account credentials.
+  # Used only for the "gcpnative" cloud provider.
+  gcpCredentialJSON: 
```

Comment:
Please use cloud storage credential provider at [gcp.go](https://github.com/milvus-io/milvus/blob/526a672bae3b08c373af989a21e46d925c051a6b/internal/storage/gcp/gcp.go). Refer to [aliyun.go](https://github.com/milvus-io/milvus/blob/526a672bae3b08c373af989a21e46d925c051a6b/internal/storage/aliyun/aliyun.go#L23)

Reply:
GCS native uses GCS credentials JSON file for authentication using GCS go package. Refer file [gcp_native_object_storage.go](https://github.com/Rijin-N/milvus/blob/9782528a6369e7f7c1e0ef690c898b694b9f2864/internal/storage/gcp_native_object_storage.go#L59).

But [gcp.go](https://github.com/milvus-io/milvus/blob/526a672bae3b08c373af989a21e46d925c051a6b/internal/storage/gcp/gcp.go) and [aliyun.go](https://github.com/milvus-io/milvus/blob/526a672bae3b08c373af989a21e46d925c051a6b/internal/storage/aliyun/aliyun.go#L23) uses S3 compatible authetication with access key and secret key using minio go package. Both are different approches.